src/components/basic/MultiSelectList/index.tsx

```diff
@@ -79,12 +81,10 @@ export const MultiSelectList = ({
 }: MultiSelectListProps) => {
   const selectHeight = popperHeight ? `${popperHeight}px` : 'auto'
   const theme = useTheme()
-  const [selected, setSelected] = useState<[]>([])
+  const [selected, setSelected] = useState<Options>([])
   const [showItems, setShowItems] = useState(false)
 
-  // Add an ESLint exception until there is a solution
-  // eslint-disable-next-line
-  const handleChange = (selectedItems: any) => {
+  const handleChange = (selectedItems: Options) => {
```

Comment:
same as above

Reply:
This change does not affect the portal app’s functionality